magicbot/magicrobot.py

```diff
@@ -386,6 +396,7 @@ def _create_components(self):
 
             # Automatically inject a logger object
             component.logger = logging.getLogger(m)
+            component.logger.addFilter(PeriodicFilter(self))
```

Comment:
Put it in the docks and then blame it on them 😂 

I'll figure out a good way to do this